package/CHANGELOG

```diff
@@ -65,9 +65,11 @@ Fixes
     argument. The directives parsed into bonds, angles, impropers, and dihedrals now
     match TPRParser. (PR #2408)
   * Added parmed to setup.py
+  * PersistenceLength.plot() now grab new axes if current axes not provided (Issue #2590)
```

Comment:
Nitpicky -- could you please change "grab" to "create"? The fixes should also be in order of most recent to least recent, i.e., your entry should be first.